src/Tasks/Microsoft.NET.Build.Tasks/targets/Microsoft.PackageDependencyResolution.targets

```diff
@@ -482,29 +481,70 @@ Copyright (c) .NET Foundation. All rights reserved.
 
   <!--
     ============================================================
-    CopyLocal Targets: For populating CopyLocal based on lock file
-     _ComputeLockFileCopyLocal
-    - ResolveLockFileCopyLocalProjectDeps
+    ComputeLockFileCopyLocal
+    Computes the files from the lock file to copy for build and publish.
     ============================================================
-    -->
+  -->
+  <Target Name="ComputeLockFileCopyLocal"
+          DependsOnTargets="_ResolveCopyLocalAssetsForBuild;
```

Comment:
It would be good to get a performance read on the cost of this, which, if I've understood correctly is going to be invoked in incremental builds where only source code has changed. 

I suspect it will be high because previously we did not even read the assets file in common incremental cases (instead only the assets.cache of ResolvePackageAssets). We may need to fold the ResolveCopyLocalAssets into ResolvePackageAssets.

cc @johnbeisner are we set up to compare a PR to baseline?

Reply:
I was thinking about combining this work into `ResolvePackageAssets` since it seems that would make sense.  I can see what I can do.

Reply:
So I've changed this a bit in the next push.  Since we're emitting all assets anyway from `ResolvePackageAssets`, I now have more metadata on the items that signify if they should be copied local as well as properties needed to satisfy the copy.  We use that to populate what should be copied local.  Thus, we still write the same number of items to the assets cache, except now with some additional metadata for items that are to be copied local for build.